01-rstudio-intro.Rmd

```diff
@@ -20,6 +20,17 @@ source("tools/chunk-options.R")
 > * Introduction to package management
 > 
 
+## Motivation
+
+Science is a multi-step process: once you've designed an experiment and collected 
+data, the real fun begins! This lesson will teach you how to start this process using 
+R and RStudio. We will begin with raw data, perform exploratory analyses, and learn 
+how to plot results graphically. This example starts with the 
+`data/gapminder-FiveYearData.csv` dataset containing population information for many 
+countries through time. Can you read the data into R? Can you plot the population for 
+Senegal? What about Croatia? By the end of these lessons you will be able to plot the 
```

Comment:
Minor change, instead of "What about Croatia?" how about "Calculate average income for countries on the asian continent?".

That covers the split-apply-combine use of plyr.